website/docs/docs-playground-app.mdx

```diff
@@ -8,7 +8,7 @@ sidebar_label: Playground app
 
 If you want to have a quick look around and test things out, you can run the playground app, bundled with this repo.
 
-1. Install dependencies via `npm install` (if you haven't already)
+1. Install dependencies via `npm install` (if you haven't already) and `cd ./playground/ios && pod install` (for iOS)
```

Comment:
Could you change it to `npm run pod-install` instead of `cd ./playground/ios && pod install`?

Reply:
Done